node/src/run_modes.rs

```diff
@@ -253,7 +254,7 @@ impl Runner for RunnerReal {
     ) -> Result<(), RunnerError> {
         self.dump_config_runner_factory
             .make()
-            .go(streams, args)
+            .go(&DirsWrapperReal, streams, args) //TODO test if changing the wrapper for the mock one will break tests
```

Comment:
Straggling TODO?

Reply:
I applied your previous comment about `.go` function and `DirsWrapper`, so this comment is probably obsolete